arbnode/batch_poster.go

```diff
@@ -763,8 +764,8 @@ func (b *BatchPoster) pollForL1PriceData(ctx context.Context) {
 					blobFeePerByte.Mul(blobFeePerByte, blobTxBlobGasPerBlob)
 					blobFeePerByte.Div(blobFeePerByte, usableBytesInBlob)
 					blobFeeGauge.Update(blobFeePerByte.Int64())
-					if l1GasPrice > blobFeePerByte.Uint64()/16 {
-						l1GasPrice = blobFeePerByte.Uint64() / 16
+					if blobGasPrice := blobFeePerByte.Uint64() / l1pricing.BatchGasUnitsPerByte; l1GasPrice > blobGasPrice {
```

Comment:
Do we use the names blob and gas interchangeably here? Do we use the word blob even if we post as calldata?